src/orders/views/OrderList/filters.ts

```diff
@@ -131,13 +70,8 @@ const _whereToLegacyVariables = (where: OrderFilterInput) => {
 export function getFilterVariables(
   params: OrderListUrlFilters,
   filterContainer: FilterContainer,
-  isFeatureFlagEnabled: boolean,
 ): OrderFilterInput {
-  let queryVariables;
-
-  if (isFeatureFlagEnabled) {
-    queryVariables = _whereToLegacyVariables(createOrderQueryVariables(filterContainer));
-  }
+  const queryVariables = _whereToLegacyVariables(createOrderQueryVariables(filterContainer));
```

Comment:
suggestion: I think since we're enabling new filtering by default we should clean up this function. We could make a ticket for it so that the scope of this PR doesn't expand